src/services/AmazonS3Service.ts

```diff
@@ -61,12 +61,16 @@ class AmazonS3Service {
           method: 'PUT',
           body: blobData,
         });
-        // Uploaded photo URL will be https://promopal.s3-us-west-1.amazonaws.com/`promotionId`.`imageType`
+        // Uploaded photo URL will be https://promopal.s3-us-west-1.amazonaws.com/`promotionId`
       })
       .catch((err: AxiosError) => {
         return Promise.reject(err);
       });
   }
+
+  getImageUrl(promotionId: string): string {
+    return `https://promopal.s3-us-west-1.amazonaws.com/${promotionId}`
+  }
 }
```

Comment:
Talked with @richardyjtian and modified the lambda function to make the `key` of the `object` just the `promotionId`. This is to support changes in the BE where do not know the image type.

@engineereng tagging you here because you're working in similar areas!